README.md

```diff
@@ -196,3 +198,8 @@ __Problem__: Messages are in the queue but not sending
 
 __Solution__: Check that `celery` is running. 
 
+---
+
+__Problem__: Running tests, `test_create_service_and_history_is_transactional` is failing with this error: `assert 'column "name" violates not-null constraint' in '(psycopg2.errors.NotNullViolation) null value in column "name" of relation "services_history" violates not-null constraint`
+
+__Solution__: Ensure your PostgreSQL version is 12.6, this error happens with version 13+.
```

Comment:
```suggestion
__Solution__: Ensure your PostgreSQL version is 11.9, this error happens with version 13+.
```

Reply:
I would like to emphasis the fact that 13+ is not supported yet, but maybe we could provide a range for that particular known error, something like: `Ensure your PostgreSQL version is above 11 but below 13.`